tests/ert_tests/models/test_base_run_model.py

```diff
@@ -54,3 +55,21 @@ def job_status(queue_index):
         jobs, status = brm.realization_progress[0][0]
         self.assertEqual(len(jobs), 1)
         self.assertIn("name", jobs[0])
+
+
+class Job:
```

Comment:
maybe `MockJob` ?

Reply:
That's better, thanks.
I've pushed a fix.